batch/batch_test.go

```diff
@@ -300,3 +300,28 @@ func TestError(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestNoWithBatching(t *testing.T) {
+	f := (&batch.Func{
+		Many: func(ctx context.Context, args []interface{}) ([]interface{}, error) {
+			return nil, nil
+		},
+	}).Invoke
+
+	ctx := context.Background()
+
+	var wg sync.WaitGroup
+	for i := 0; i < 20; i++ {
```

Comment:
i think you don't need this for loop wait group contraption, since the panic we're testing is immediate when f is invoked

Reply:
👍  removed it